client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/sidebar.tsx

```diff
@@ -101,10 +101,11 @@ const Sidebar = ( { sidebarDomain, siteSlug, submit, goToStep, flow }: SidebarPr
 
 	const { title, launchTitle, subtitle } = getLaunchpadTranslations( flow );
 
-	const { getPlanCartItem, getDomainCartItem } = useSelect(
+	const { planCartItem, domainCartItem, productCartItems } = useSelect(
```

Comment:
**Note:**

I made the switch from returning selectors in this `useSelect` call to returning derived data ( Ex. getPlanCartItem -> getPlanCartItem() ) because we only need the data in this component.

From my understanding, the only time we'd really need access to the selectors is if we have some sort of event callback where we want to invoke them with dynamic arguments